src/gpdevboard/usb.cpp

```diff
@@ -26,6 +26,11 @@
 #include <gpdevboard.h>
 #include <unistd.h>
 
+#if defined(__APPLE__)
+	#include <cstdlib>
```

Comment:
<cstdlib> is always supposed to be included when calling `wcstombs`. It only happens to break on macOS due to transitive dependencies in other headers. Likewise, <cstring> is always supposed to be included when calling `memcpy` (in my hack). These shouldn't be in the __APPLE__ guard.